DepositeCalcTests/Pages/LoginPage.cs

```diff
@@ -13,8 +13,8 @@ public LoginPage(IWebDriver driver) : base(driver)
         private IWebElement LoginFld => driver.FindElement(By.XPath("//th[text()='User:']/..//input"));
         private IWebElement LoginBtn => driver.FindElement(By.XPath("//button[@id='loginBtn']"));
         private IWebElement PassworldFld => driver.FindElement(By.XPath("//th[text()='Password:']/..//input"));
-        private IWebElement RemindPasswordBtn => driver.FindElement(By.XPath("//button[@id='remindBtn']"));
-        public RemindPasswordView RemindPasswordForm => new RemindPasswordView(driver, RemindPasswordBtn);
```

Comment:
Please restore these lines.

Reply:
@varajan If these lines are not in use, why are they needed here?